v2.0/grant-roles.md

```diff
@@ -0,0 +1,88 @@
+---
+title: GRANT &lt;roles&gt;
+summary: The GRANT <roles> statement grants user privileges for interacting with specific databases and tables.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> The `GRANT <roles>` [statement](sql-statements.html) lets you add a [role](roles.html) or [user](create-and-manage-users.html) as a member to a role.
+
+{{site.data.alerts.callout_info}}You must have an <a href="enterprise-licensing.html">enterprise license</a> to add a member to a role.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Synopsis
+
+<section>{% include sql/{{ page.version.version }}/diagrams/grant_roles.html %}</section>
+
+## Required Privileges
+
+The user granting role membership must be a role admin (i.e., members with the `ADMIN OPTION`) or a superuser (i.e., a member of the `admin` role).
+
+## Considerations
+
+- Users and roles can be members of roles.
+- The `root` user is automatically created as an `admin` role and assigned the `ALL` privilege for new databases.
+- All privileges of a role are inherited by all its members.
```

Comment:
It would be good to re-iterate the "no membership cycle" rule, this is where it's applied.